### PR TITLE
Added baked potato recipe

### PR DESCRIPTION
**Pull request by teroff**

### CODE REVIEW

File: src/baked-potatoes.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Baked Potatoes
+
+- ⏲️ Prep time: 10 minutes
+- 🍳 Cook time: 20-25 minutes
+- 🍽️ Servings: 4-6 people
+
+## Ingredients
+
+- 6-7 Yukon pototoes (you can use any potatoes you like)
```
Comment (hallzy) [inline src/baked-potatoes.md:9]:
```suggestion
- 6-7 Yukon potatoes (you can use any potatoes you like)
```

File: src/baked-potatoes.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- 1/3 cup of any vegitable oil
+- 1 teaspoon salt
+- Black pepper
+- Any seasoning you like
```
Comment (hallzy) [inline src/baked-potatoes.md:13]:
```suggestion
```

Remove salt/pepper and seasonings 'to taste' from ingredients. They can be mentioned below in the directions.

File: src/baked-potatoes.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Baked Potatoes
+
+- ⏲️ Prep time: 10 minutes
+- 🍳 Cook time: 20-25 minutes
+- 🍽️ Servings: 4-6 people
+
+## Ingredients
+
+- 6-7 Yukon pototoes (you can use any potatoes you like)
+- 1/3 cup of any vegitable oil
```
Comment (hallzy) [inline src/baked-potatoes.md:10]:
```suggestion
- 1/3 cup of any vegetable oil
```

File: src/baked-potatoes.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- 1 teaspoon salt
+- Black pepper
+- Any seasoning you like
+
+## Directions
+1. Preheat the oven to 425F/220C
```
Comment (hallzy) [inline src/baked-potatoes.md:16]:
```suggestion
1. Preheat the oven to 425°F/220°C
```

File: src/baked-potatoes.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+2. Wash potatoes and slice into 5mm slices
+3. Put the sliced poptatoes into a bowl
```
Comment (hallzy) [inline src/baked-potatoes.md:18]:
```suggestion
3. Put the sliced potatoes into a bowl
```

File: src/baked-potatoes.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+- 6-7 Yukon pototoes (you can use any potatoes you like)
+- 1/3 cup of any vegitable oil
+- 1 teaspoon salt
+- Black pepper
+- Any seasoning you like
+
+## Directions
+1. Preheat the oven to 425F/220C
+2. Wash potatoes and slice into 5mm slices
```
Comment (hallzy) [inline src/baked-potatoes.md:17]:
```suggestion
2. Wash potatoes and cut into into 5mm slices
```

File: src/baked-potatoes.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+3. Put the sliced poptatoes into a bowl
+4. Add salt, pepper and any other seasoning you like
+5. Add 1/3 cup of a vegitable oil
```
Comment (hallzy) [inline src/baked-potatoes.md:20]:
```suggestion
5. Add 1/3 cup of a vegetable oil
```

File: src/baked-potatoes.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+6. Mix everything and put it on the baking pan
```
Comment (hallzy) [inline src/baked-potatoes.md:21]:
```suggestion
6. Mix everything and put it on a baking sheet (optionally lined with baking paper to make cleanup easier) for 20 to 25 minutes.
```

File: src/baked-potatoes.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+I preffer to use baking paper, it makes everything cleaner and easier. Put your potatoes on the
+baking pan and put it on the oven for 20-25 minutes.
+
```
Comment (hallzy) [inline src/baked-potatoes.md:25]:
```suggestion
```

**Review by hallzy — CHANGES_REQUESTED** [review]

File: src/baked-potatoes.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+2. Wash potatoes and cut into into 5mm slices
+3. Put the sliced potatoes into a bowl
+4. Add salt, pepper and any other seasoning you like
+5. Add 1/3 cup of a vegetable oil
+6. Add salt, pepper and any other seasonings you like
+7. Mix everything and put it on a baking sheet (optionally lined with baking paper to make cleanup easier) for 20 to 25 minutes.
+
+I preffer to use baking paper, it makes everything cleaner and easier. Put your potatoes on the baking pan and put it on the oven for 20-25 minutes.
+
+## Contribution
```
Comment (hallzy) [inline src/baked-potatoes.md:23]:
```suggestion
## Contribution
```

Can probably get rid of this since it is in step 7 now.

**Review by hallzy — CHANGES_REQUESTED** [review]

**Review by hallzy — APPROVED** [review]

**Review by siggines — CHANGES_REQUESTED** [review]

**Review by siggines — APPROVED** [review]
All good 👌

**Review by DGKSK8LIFE — APPROVED** [review]

**Review by AndrewDaGuy — APPROVED** [review]

**Pull request closed without merging**